client/dashboard/components/date-range-picker/date-inputs.tsx

```diff
@@ -43,7 +45,11 @@ export function DateInputs( {
 					type="date"
 					label={ __( 'Start date' ) }
 					value={ fromStr }
-					onChange={ ( value?: string ) => onFromChange( value ?? '' ) }
+					onFocus={ () => onInteract?.() }
```

Comment:
Does it make sense from an API perspective to export the `onFocus` events, the same way we do for the `onChange` functions, instead of a custom `onInteract` callback?

Reply:
I was thinking that since DataInputs calls `onInteract` in the `onChange` and `onFocus` functions, we wouldn't need to have `onInteract` if we exposed `onFocus` as well. Seeing that the events (blur/focus) are now exposed, is `onInteract` still needed?